action.yml

```diff
@@ -26,11 +27,34 @@ runs:
     env:
       INPUT_APP: ${{ inputs.app_name }}
       INPUT_TUTOR_VERSION: ${{ inputs.tutor_version }}
+  
+  - name: Prepare Tutor Nightly and Launch
+    if: ${{ inputs.tutor_version == 'nightly' }}
+    run: |
+      pip install virtualenv
+      virtualenv venv
+      source venv/bin/activate
+      git clone --branch=nightly --depth=1 https://github.com/overhangio/tutor.git
+      chmod 777 tutor -R
+      pip install -e "./tutor[full]"
+      TUTOR_ROOT="$(pwd)" tutor --version
+      TUTOR_ROOT="$(pwd)" tutor config save
+      TUTOR_ROOT="$(pwd)" tutor mounts add lms,cms,lms-worker,cms-worker:$(pwd)/$INPUT_APP:/openedx/$INPUT_APP
+      chmod 777 . -R
+      TUTOR_ROOT="$(pwd)" tutor images build openedx 
```

Comment:
I think that only with the `launch` command should it work, can we try?

Reply:
I could try this one.

Reply:
It would be great if it works... some lines could be unified in another step `Setup and Launch`

Reply:
I test it without building the openedx image and fail https://github.com/eduNEXT/eox-tenant/actions/runs/10826637377/job/30246102551?pr=216 

Can not install the test requirements.